typing/typedecl.ml

```diff
@@ -4483,15 +4494,19 @@ let report_error ppf = function
         Style.inline_code "[@@untagged]"
   | Cannot_unbox_or_untag_type Unboxed ->
       fprintf ppf "@[Don't know how to unbox this type.@ \
-                   Only %a, %a, %a, %a, vector primitives, and@ \
+                   Only %a, %a, %a, %a, %a, %a, vector primitives, and@ \
```

Comment:
I think we should pick one of either unboxed or untagged for int8/int16. Untagged seems like a better fit.